packages/reference/src/common/useAccessApi.ts

```diff
@@ -7,15 +7,18 @@ import type {
 import type { Entry } from '@contentful/field-editor-shared';
 
 function makeEntryFromType(contentTypeId: string) {
-  return {
+  return ({
     sys: {
       type: 'Entry',
       contentType: {
         sys: { id: contentTypeId },
       },
     },
+    metadata: {
+      tags: [],
+    },
     fields: {},
-  } as Entry;
+  } as unknown) as Entry;
```

Comment:
I wonder if we should add `metadata: {tags: []}` to the [Entry](https://github.com/contentful/field-editors/blob/master/packages/_shared/src/typesEntity.ts) type?